aggregation/simple/src/test/java/com/spotify/heroic/aggregation/simple/DeltaTest.java

```diff
@@ -0,0 +1,49 @@
+package com.spotify.heroic.aggregation.simple;
+
+import com.spotify.heroic.aggregation.AggregationSession;
+import com.spotify.heroic.common.DateRange;
+import com.spotify.heroic.metric.Point;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+
+public class DeltaTest {
```

Comment:
Good stuff, but consider adding a test to [AbstractClusterQueryIT](https://github.com/spotify/heroic/blob/master/heroic-dist/src/test/java/com/spotify/heroic/AbstractClusterQueryIT.java), that will make sure your aggregation works as expected over a cluster (i.e. it is not distributable).